newIDE/app/src/ProjectsStorage/LocalFileStorageProvider/LocalProjectWriter.js

```diff
@@ -24,6 +24,34 @@ export const splittedProjectFolderNames = [
   'eventsFunctionsExtensions',
 ];
 
+const deleteExistingFilesFromDirs = (
+  project: gdProject,
+  projectPath: string
+) => {
+  const entries = fs.readdirSync(projectPath);
+
+  if (project.isFolderProject()) {
+    //If multiFile enabled in settings and directories exist!
+    entries.forEach(entry => {
+      let dirPath = projectPath.concat('\\', entry);
```

Comment:
I forgot to check how you handle paths: Using strings that you concat yourself won't work because Windows user `\` separator while Linux and macOS user `/`. So you should use `path.join` instead. You can find other users of path in the codebase.

Reply:
Furthermore, you can use `const` instead of `let` since you never reassign the variable value